iOSClient/Main/Create cloud/NCCreateFormUploadConflict.swift

```diff
@@ -290,30 +251,32 @@ extension NCCreateFormUploadConflictDelegate {
             if metadatasConflictNewFiles.contains(metadata.ocId) && metadatasConflictAlreadyExistingFiles.contains(metadata.ocId) {
```

Comment:
I'm not sure about the difference between `metadatasConflictNewFiles` and `metadatasConflictAlreadyExistingFiles`. Would you mind to explain what they are used for? Or better, add documentation for those.